### PR TITLE
Update Japanese translations

### PR DESCRIPTION
**Pull request by eve0415**
Fix and update Japanese translations.

### CODE REVIEW

**Comment by CharlieWhile13** [comment]
Could you change the branch you're merging into to amy?

**Comment by eve0415** [comment]
> Could you change the branch you're merging into to amy?

Done

File: Sileo/Sileo/ja.lproj/Localizable.strings [hunk 1]
```diff
@@ -1,102 +1,168 @@
...
+"Settings_Payment_Provider.Purchases_Heading" = "購入済みパッケージ";
+"Settings_Payment_Provider.No_Purchases" = "購入したパッケージはありません。";
+
+"Settings" = "設定";
+"Settings_Page_Title" = "設定";
+"Dark_Mode_Toggle" = "ダークモード";
+"Theme" = "テーマ";
+"Tint_Color" = "色合い";
+"Reset_Tint_Color" = "色合いをリセット";
+"Auto_Complete_Queue" = "キューをオートコンプリート";
```
Comment (sohsatoh) [inline Sileo/Sileo/ja.lproj/Localizable.strings:129]: This is a switch, so it should be "自動完了キュー" or "オートコンプリートキュー"

**Review by sohsatoh — CHANGES_REQUESTED** [review]
Reviewed (except for stringsdict)

**Review by sohsatoh — CHANGES_REQUESTED** [review]
About stringsdict, you need to make the following changes.

- "クリップボードで見つかった %#@urls@ を追加しますか" to "クリップボードから %#@urls@ を追加しますか"
- "No Packages" to "パッケージ無し"

**Comment by eve0415** [comment]
Applied all suggestions.

**Comment by sohsatoh** [comment]
@eve0415 i will check after the graduation ceremony. (by the way, we gonna the same university. you know?)

**Comment by eve0415** [comment]
Yeah, I've realized that a while ago; Congrat on your graduation

**Comment by sohsatoh** [comment]
@eve0415 Thanks, good luck with your studies and enjoy your uni life!

**Comment by CharlieWhile13** [comment]
You now have the award for the most wholesome pull request on Sileo

**Review by sohsatoh — APPROVED** [review]

**Pull request merged**